### PR TITLE
Add redis payloads

### PR DESCRIPTION
**Pull request by haddasbronfman**

### CODE REVIEW

File: src/instrumentations/index.ts [hunk 1]
```diff
@@ -18,8 +18,10 @@ import { Instrumentation } from '@opentelemetry/instrumentation';
 import { Options } from '../options';
 import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
 import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
-import { configureHttpInstrumentation } from './extentions/http';
 import { diag } from '@opentelemetry/api';
+import { configureHttpInstrumentation } from './extentions/http';
+import { configureAWSInstrumentation } from './extentions/aws/aws_sdk';
+import { configureRedisnstrumentation } from './extentions/redis';
```
Comment (arbiv) [inline src/instrumentations/index.ts:24]: please fix typo

File: src/instrumentations/index.ts [hunk 2]
```diff
@@ -18,8 +18,10 @@ import { Instrumentation } from '@opentelemetry/instrumentation';
...
 
 export function getInstrumentations(options: Options): Instrumentation[] {
   const instrumentations = getNodeAutoInstrumentations();
```
Comment (arbiv) [inline src/instrumentations/index.ts:27]: shouldn't we push also redis instrumentation here?

**Review by arbiv — CHANGES_REQUESTED** [review]

File: src/instrumentations/index.ts (hunk 2 above)
Comment (haddasbronfman) [reply src/instrumentations/index.ts:27]: The redis instrumentation is in: otel-js/src/instrumentations/extentions/redis.ts - this file creates the "configureRedisnstrumentation" in which we use here in index.ts

File: src/instrumentations/index.ts (hunk 1 above)
Comment (haddasbronfman) [reply src/instrumentations/index.ts:24]: done

**Review by arbiv — APPROVED** [review]

**Pull request merged**